_data/social.yml

```diff
@@ -285,8 +285,10 @@ websites:
     - name: XING
       url: https://www.xing.com
       img: xing.png
-      tfa: No
-      twitter: XING_com
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      doc: https://www.xing.com/settings/account/credentials/twofa
```

Comment:
```suggestion

```
We don't list doc links that's behind a login screen. It also seems to just be the settings page and not a documentation.
Could you provide us with a screenshot of the login process or the settings page? (With any personal information blurred out)

Reply:
All the "documentation" you'll get from XING is written on that page. It's actually quiet empty, see for yourself:

![screenshot-www xing com-2018 10 24-20-45-40](https://user-images.githubusercontent.com/86269/47453805-e67cb480-d7cd-11e8-81ad-f81f5d076c88.png)

Clicking on "Learn more about two-factor authentication" leads to the following modal:

![screenshot-www xing com-2018 10 24-20-46-01](https://user-images.githubusercontent.com/86269/47453898-188e1680-d7ce-11e8-91fd-bf9404ef25f9.png)

Reply:
Alright, that looks good. Just remove the doc link and the pull request should be good to go. 😄